Containerfile

```diff
@@ -0,0 +1,51 @@
+# This file will contain the configuration for containerized esi-leap
+
+# Use python as the parent image for esi-leap
+
+FROM python:3.11
+
+# Use label descriptor
+
+LABEL description="ESI-LEAP container file"
+
+# Maintainer details
+
+MAINTAINER <tzumainn@redhat.com>
+
+# The working directory for esi-leap inside the container
+
+WORKDIR /esi-leap
+
+# Copy the files of esi leap inside the container
+# Syntax : COPY <src> <destination>
+
+COPY run_services.sh run_services.sh
+
+# Please make sure to have esi-leap.conf file in same directory as containerfile 
+# or make sure to mention the right path of the file below.
+
+#COPY esi-leap.conf /etc/esi-leap/esi-leap.conf
+
+# Run command to install dependencies for esi-leap
+
+RUN pip install esi-leap
+
+RUN pip install mysql-connector
+
+RUN mkdir -p /var/log/esi-leap
+
+RUN touch /var/log/esi-leap/esi-leap-dbsync.log
+
+RUN dnf update -y
+
+RUN dnf install python3-pymysql
```

Comment:
Do the Python container images ship with a Fedora-based base OS? I thought they were Debian-based (and if they are these lines should be replaced with equivalent `apt` commands).